201-vnet-to-vnet-peering/azuredeploy.json

```diff
@@ -2,28 +2,48 @@
     "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "vNet1Name": {
+            "type":"string",
+            "metadata": {
+                "description": "Name for vNet 1"
+            }
+        },
+        "vNet2Name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name for vNet 2"
+            }
+        },
+        "vNet1tovNet2PeeringName": {
```

Comment:
Can we concat a name for these?  e.g. concat(parameters('vnet1Name'), '-', parameters('vnet2Name') - otherwise users are likely to add chars not allowed.